pkgs/data/fonts/dm-sans/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenvNoCC, lib, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
We are not compiling anything so we can use fetchzip and postFetch

Reply:
Thanks, will fix later today.

Reply:
FYI, You can using `fetchzip` like this:
https://github.com/zendo/nixpkgs/blob/703878418fd940d0770954ba8188f4184340f66f/pkgs/data/fonts/lxgw-wenkai/default.nix